capitolwords_ng/capitolweb/legislators/models.py

```diff
@@ -43,11 +49,20 @@ class Term(models.Model):
     rss_url = models.URLField
     url = models.URLField()
 
+    class Meta:
+        ordering = ('start_date',)
+
 
 class ExternalId(models.Model):
     person = models.ForeignKey(CongressPerson, on_delete=models.CASCADE, related_name='external_ids')
     type = models.CharField(max_length=50)
     value = models.CharField(max_length=50)
 
 
+def get_current_legislators(state=None):
+    terms = Term.objects.filter(start_date__lte=date.today(), end_date__gte=date.today())
+    if state:
+        terms = Term.objects.filter(start_date__lte=date.today(), end_date__gte=date.today(), state=state)
+    return [term.person for term in terms]
```

Comment:
This endpoint seems super useful. This code will cause Django to issue a separate db query for each `term.person`, though. Probably better to do something like `CongressPerson.objects.filter(Q(terms__start_date__lte=date.today(), terms__end_date__gte=date.today()))` which will do it in a single db query.